### PR TITLE
feature/OC-662/Metadata-tab

### PR DESCRIPTION
**Pull request by remko48**

### CODE REVIEW

File: pwa/src/templates/templateParts/schemaForm/SchemaFormTemplate.tsx [hunk 1]
```diff
@@ -57,6 +57,8 @@ export const SchemaFormTemplate: React.FC<SchemaFormTemplateProps & ReactHookFor
     setSimpleProperties([]);
     setComplexProperties([]);
 
+    if (!schema.properties) return;
```
Comment (lencodes) [inline pwa/src/templates/templateParts/schemaForm/SchemaFormTemplate.tsx:60]: Is this intentional? Just checking 👍🏼

File: pwa/src/templates/objectDetailTemplate/ObjectDetailTemplate.tsx [hunk 2]
```diff
@@ -70,12 +71,65 @@ export const ObjectDetailTemplate: React.FC<ObjectDetailTemplateProps> = ({ obje
...
+                      <span onClick={() => navigate(`/schemas/${getObject.data._self?.schema.id}`)}>
+                        <Link icon={<FontAwesomeIcon icon={faArrowRight} />} iconAlign="start">
+                          {getObject.data._self?.schema.name ?? "-"}
+                        </Link>
+                      </span>
+                    </TableCell>
+                  </TableRow>
+                </TableBody>
+              </Table>
+            )}
```
Comment (lencodes) [inline pwa/src/templates/objectDetailTemplate/ObjectDetailTemplate.tsx:128]: I think a vertical table would be better suited here (there's no way that this table would ever have a second row 👍🏼)

**Review by lencodes — CHANGES_REQUESTED** [review]
Looks great. Left one comment.

File: pwa/src/templates/templateParts/schemaForm/SchemaFormTemplate.tsx (hunk 1 above)
Comment (remko48) [reply pwa/src/templates/templateParts/schemaForm/SchemaFormTemplate.tsx:60]: It is, without this line objects with no properties cannot be displayed and will cause an error.
And we still want to display an object even if the object does not have any properties.

**Review by lencodes — APPROVED** [review]
Nice! 👍🏼

**Pull request merged**